src/AppBundle/Resources/views/Layouts/moj_template.html.twig

```diff
@@ -63,103 +63,88 @@
 
     <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
 
-    <div id="skiplink-container">
-      <div>
-        <a href="#content" class="skiplink">Skip to main content</a>
-      </div>
-    </div>
-
-    <div id="global-cookie-message">
-      <div class="outer-block">
-        <div class="inner-block">
-
+    <a href="#content" class="govuk-skip-link">Skip to main content</a>
 
+    <div class="opg-cookie-banner">
         {% block cookieMessage %}{% endblock %}
-
-
-        </div>
-      </div>
     </div>
-    <!--end global-cookie-message-->
 
 
-    <header id="global-header" class="with-proposition {% block headerClass %}{% endblock %}">
-      <div class="header-wrapper {% if env == 'admin' %}wider{% endif %}">
-        <div class="header-global">
-          <div class="header-logo">
-            <a href="https://www.gov.uk/" title="Go to the GOV.UK homepage" id="logo" class="content">
-              <img src="/images/gov.uk_logotype_crown.png" width="35" height="31" alt=""> GOV.UK
+    <header class="govuk-header" role="banner" data-module="header">
+      <div class="govuk-header__container govuk-width-container {% if env == 'admin' %}opg-width-container--wider{% endif %}">
+        <div class="govuk-header__logo">
+            <a href="#" class="govuk-header__link govuk-header__link--homepage">
```

Comment:
<a href="#"> presumably not intentional? Presumably should link to gov.uk given the text inside?

Reply:
Quite right; fixed.